pkg/tools/backplanecli/backplanecli.go

```diff
@@ -36,42 +35,17 @@ func (t *Tool) Install() error {
 		return err
 	}
 
-	// Determine which assets to download
-	var checksumAsset *gogithub.ReleaseAsset
-	var backplaneArchiveAsset *gogithub.ReleaseAsset
-	arch := runtime.GOARCH
-	if arch == "amd64" {
-		arch = "x86_64"
+	matches := github.FindAssetsForArchAndOS(release.Assets)
+	if len(matches) != 1 {
+		return fmt.Errorf("unexpected number of assets found matching system spec: expected 1, got %d.\nMatching assets: %v", len(matches), matches)
 	}
-	for _, asset := range release.Assets {
-		if asset.GetName() == "checksums.txt" {
-			if checksumAsset.GetName() != "" {
-				return fmt.Errorf("detected duplicate backplane-cli checksum assets")
-			}
-			checksumAsset = asset
-			continue
-		}
-		// Exclude assets that do not match system architecture
-		if !strings.Contains(asset.GetName(), arch) {
-			continue
-		}
-		// Exclude assets that do not match system OS
-		if !strings.Contains(strings.ToLower(asset.GetName()), strings.ToLower(runtime.GOOS)) {
-			continue
-		}
+	backplaneArchiveAsset := matches[0]
 
-		if backplaneArchiveAsset.GetName() != "" {
-			return fmt.Errorf("detected duplicate backplane-cli binary assets")
-		}
-		backplaneArchiveAsset = asset
-	}
-	// Ensure both checksum and binary were retrieved
-	if backplaneArchiveAsset.GetName() == "" {
-		return fmt.Errorf("failed to find valid backplane-cli binary asset")
-	}
-	if checksumAsset.GetName() == "" {
-		return fmt.Errorf("failed to find valid backplane-cli checksum asset")
+	matches = github.FindAssetsContaining([]string{"checksums.txt"}, release.Assets)
```

Comment:
I realize these are too-specific files for each tool, but would it make sense to store the tool-specific values in a single map of tools somewhere centrally instead of setting the values  hard-coded like it is here?

The simplification of the FindAssets* stuff might negate the need for segregated logic for each tool?